BusinessLogic/MapperProfile/MapperProfile.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ApiModels.Models;
+using AutoMapper;
+using DataAccess.Context;
+using Domain.EF_Models;
+
+using System.Linq;
+
+namespace TimeOffTracker.WebApi.MapperProfile
+{
+    public class MapperProfile : Profile
+    {
+        private TimeOffTrackerContext _context;
+
+        public MapperProfile(TimeOffTrackerContext context)
+        {
+            _context = context;
+            CreateMap<User, UserApiModel>()
+                .ForMember(model => model.Role, opt => opt.MapFrom(usr =>
```

Comment:
It's business logic part. 
The right way to do this

1. Create a UserService class in BusinessLogic/Services/
2. Inject IRepository<User> and IMapper to consctuctor
3. Add Method IEnumeralbe<UserApiModel> GetUsers - mapper.Map<UserApiModel>(repository.GetAll()).ToList()
4. Call GetUsers  in the controller

Reply:
Done